main.go

```diff
@@ -105,8 +103,7 @@ func main() {
 	ctx := context.NewContext()
 	err = ctx.LoadConfig(configurationFile)
 	if err != nil {
-		fmt.Println("Loading configuration file failed.")
-		return
+		log.Fatalln("Loading configuration file failed.")
```

Comment:
The reason for using fmt was to not show the user a log line on stdout. With log.Fatal the uers gets `FATA[0000] Loading configuration file failed.` What looks kind of odd to me.

Reply:
At least it's consistent with the rest of the output (if any).

But if you want it differently, we can simply replace the `return` with `os.Exit(1)`.

Reply:
Yes, I would prefer os.Exit here. The difference is that the config test is usually issued manually by the user, the output is presented directly to the console. The rest of the Sidecar runs typically as a background daemon, so log files with a proper log format are the way to print stuff.

Reply:
> The difference is that the config test is usually issued manually by the user, the output is presented directly to the console.

I'm not sure this is often the case.

When using configuration management (or even in init scripts), a validation of the configuration file is performed before the service is restarted, so that an invalid configuration wouldn't lead to service downtime.